obal/__init__.py

```diff
@@ -46,12 +46,7 @@ def data_path():
         """
         path = os.environ.get('OBAL_DATA')
         if path is None:
-            path = pkg_resources.resource_filename(__name__, 'data')
-            if not os.path.isabs(path):
-                # this is essentially a workaround for
-                # https://github.com/pytest-dev/pytest-xdist/issues/414
-                distribution = pkg_resources.get_distribution('obal')
-                path = os.path.join(distribution.location, path)
+            path = str(resources.files(__name__) / 'data')
```

Comment:
Note to self: added in Python 3.9 (https://docs.python.org/3/library/importlib.resources.html#importlib.resources.files). This call looks correct for both 3.9 and 3.12+ (where it was changed).